apps/ca.c

```diff
@@ -238,7 +238,7 @@ OPTIONS ca_options[] = {
      "msie modifications to handle all those universal strings"},
     {"crldays", OPT_CRLDAYS, 'p', "Days is when the next CRL is due"},
     {"crlhours", OPT_CRLHOURS, 'p', "Hours is when the next CRL is due"},
-    {"crlsec", OPT_CRLSEC, 'p'},
+    {"crlsec", OPT_CRLSEC, 'p', "Seconds is when the next CRL is due"},
```

Comment:
Seconds *until" the next CRL is due? Also probably should be the same for crldays and crlhours.